lanGhost.py

```diff
@@ -34,11 +35,7 @@
 
 def refreshNetworkInfo():
     try:
-        global iface_mac
-        global ip_range
-        global gw_ip
-        global gw_mac
-        global ip
+        global iface_mac, p_range, w_ip, w_mac, p
```

Comment:
First letters are missing. Could you fix it?
Also please use the "🎨' emoji at the beginning of the commit.
Thanks Kama, but this pull request is getting way too complicated for what it is :D

Reply:
Woops.. (facepalm)